py/py.mk

```diff
@@ -278,7 +278,10 @@ endif
 
 # Sources that may contain qstrings
 SRC_QSTR_IGNORE = py/nlr%
+SRC_QSTR_EMITNATIVE = py/emitn%
 SRC_QSTR = $(SRC_MOD) $(filter-out $(SRC_QSTR_IGNORE),$(PY_CORE_O_BASENAME:.o=.c)) $(PY_EXTMOD_O_BASENAME:.o=.c)
+# Sources that only hold QSTRs after pre-processing.
```

Comment:
OK, I see the comment here, but maybe rename it to `SRC_QSTR_PREPROCESSOR` or `SRC_QSTR_CPP`.

Reply:
Done!